pre_commit/constants.py

```diff
@@ -14,13 +14,13 @@
 STAGES = (
     'commit', 'merge-commit', 'prepare-commit-msg', 'commit-msg',
     'post-commit', 'manual', 'post-checkout', 'push', 'post-merge',
-    'post-rewrite',
+    'post-rewrite', 'rebase',
 )
 
 HOOK_TYPES = (
     'pre-commit', 'pre-merge-commit', 'pre-push', 'prepare-commit-msg',
     'commit-msg', 'post-commit', 'post-checkout', 'post-merge',
-    'post-rewrite',
+    'post-rewrite', 'rebase',
```

Comment:
this should be called `pre-rebase` -- the original `commit` stage is a mistake that's too late to fix now

Reply:
I see, thank you! Applied!